lint failed

> Task :react-native-webview:lint FAILED
Ran lint on variant release: 11 issues found
Ran lint on variant debug: 11 issues found
Wrote HTML report to file:///Users/vinurs/system-configuration/work/glm/GlmApp/node_modules/react-native-webview/android/build/reports/lint-results.html
Wrote XML report to file:///Users/vinurs/system-configuration/work/glm/GlmApp/node_modules/react-native-webview/android/build/reports/lint-results.xml

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':react-native-webview:lint'.
> Lint found errors in the project; aborting build.

  Fix the issues identified by lint, or add the following to your build script to proceed with errors:
  ...
  android {
      lintOptions {
          abortOnError false
      }
  }
  ...

  The first 3 errors (out of 4) were:
  /Users/vinurs/system-configuration/work/glm/GlmApp/node_modules/react-native-webview/android/src/main/java/com/reactnativecommunity/webview/RNCWebViewManager.java:369: Error: Call requires API level 21 (current min is 16): android.webkit.CookieManager#removeAllCookies [NewApi]
      CookieManager.getInstance().removeAllCookies(null);
                                  ~~~~~~~~~~~~~~~~
  /Users/vinurs/system-configuration/work/glm/GlmApp/node_modules/react-native-webview/android/src/main/java/com/reactnativecommunity/webview/RNCWebViewManager.java:780: Error: Call requires API level 21 (current min is 16): android.webkit.PermissionRequest#getResources [NewApi]
        String[] requestedResources = request.getResources();
                                              ~~~~~~~~~~~~
  /Users/vinurs/system-configuration/work/glm/GlmApp/node_modules/react-native-webview/android/src/main/java/com/reactnativecommunity/webview/RNCWebViewManager.java:804: Error: Call requires API level 21 (current min is 16): android.webkit.PermissionRequest#deny [NewApi]
          request.deny();
                  ~~~~

--- a/android/src/main/java/com/reactnativecommunity/webview/RNCWebViewManager.java
+++ b/android/src/main/java/com/reactnativecommunity/webview/RNCWebViewManager.java
@@ -837,6 +837,7 @@ public class RNCWebViewManager extends SimpleViewManager<WebView> {
     }
 
     // Fix WebRTC permission request error.
+    @TargetApi(Build.VERSION_CODES.LOLLIPOP)
     @Override
     public void onPermissionRequest(final PermissionRequest request) {
       String[] requestedResources = request.getResources();

